Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/ObjectBinding.Web.Development.WebTesting.IntegrationTests.csproj

```diff
@@ -14,6 +14,10 @@
     <Compile Include="..\..\..\SharedSource\Development\NUnit\UnitTesting\*.cs" Link="SharedSource\NUnit\UnitTesting\%(Filename)%(Extension)" />
     <EmbeddedResource Include="SavedTestScreenshots\**\*.png" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\..\..\SharedSource\Core\Utilities\Assertion.cs" Link="SharedSource\Utilites\%(Filename)%(Extension)" />
+    <Compile Include="..\..\..\SharedSource\Core\RoslynAnnotations\NullableAnnotations.cs" Link="SharedSource\RoslynAnnotations\NullableAnnotations.cs" />
```

Comment:
Usually, we have all shared source imports in the same location so I would move those two lines before line 14.

If you want you can do a small cleanup an put the `EmbeddedResource` into its own `ItemGroup`. Usually an `Itemgroup` will only contains items of the same type.